commands/release.py

```diff
@@ -1,19 +1,40 @@
 import click
 import requests
 
-from services.releaser import get_releaser_url
+from data.Releaser import ReleaseRankmiResponse, ReleaseProjectResponse
+from services.releaser import get_releaser_url, rel_rankmi_response, rel_project_response
+from services.youtrackservice import get_youtrack_user
```

Comment:
por qué en algunos se usa piso y en los otros no?

Reply:
Y por qué hay algunos con mayúscula y otros no?

Reply:
Convención de Python
https://www.python.org/dev/peps/pep-0008/#class-names
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names